### PR TITLE
feature/COR-1676-consistent-hospital-pages

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard29**
## Summary
 
* Added bandpadding as parameter
* Added bars in hospital pages
 
#### Before
<details>
<summary>Toggle before screenshots</summary>
 
image
image

</details>
 
#### After
<details>
<summary>Toggle after screenshots</summary>
 
image
image

</details>

### CODE REVIEW

File: packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx [hunk 1]
```diff
@@ -156,110 +156,111 @@ const HospitalsAndCarePage = (props: StaticProps<typeof getStaticProps>) => {
...
-              toggle={{
-                initialValue: selectedBedsOccupiedOverTimeChart,
-                items: bedsOccupiedOverTimeToggleItems,
-                onChange: (value) => setSelectedBedsOccupiedOverTimeChart(value),
-              }}
-            >
-              <TimeSeriesChart
-                accessibility={{
-                  key: 'hospital_beds_occupied_over_time_chart',
+          <InView rootMargin="400px">
```
Comment (VWSCoronaDashboard30) [inline packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx:159]: I believe this is a magic number

File: packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx [hunk 2]
```diff
@@ -156,110 +156,111 @@ const HospitalsAndCarePage = (props: StaticProps<typeof getStaticProps>) => {
...
-                }}
-              />
-            </ChartTile>
-          )}
+                    {
+                      type: 'bar',
+                      metricProperty: 'beds_occupied_covid',
+                      label: textNl.icu.chart_beds_occupied.legend_trend_label,
+                      color: colors.primary,
+                      bandPadding: 0.4,
```
Comment (VWSCoronaDashboard30) [inline packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx:245]: Can you extract this as a constant?

File: packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx [hunk 3]
```diff
@@ -312,6 +313,7 @@ const HospitalsAndCarePage = (props: StaticProps<typeof getStaticProps>) => {
                     metricProperty: 'influx_covid_patients',
                     label: textNl.hospitals.chart_patient_influx.legend_title_trend_label,
                     color: colors.primary,
+                    bandPadding: 0.4,
```
Comment (VWSCoronaDashboard30) [inline packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx:316]: same here with the magic number

File: packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx (hunk 1 above)
Comment (VWSCoronaDashboard29) [reply packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx:159]: True, but it's done the same way in many different places as well.
See:

- [code]\de-coronaprik.tsx:197
- [code]\positieve-testen.tsx:214
- [code]\rioolwater.tsx:184
- [code]\sterfte.tsx:179

How shall we pick this up further?

File: packages/app/src/components/time-series-chart/components/series.tsx [hunk 4]
```diff
@@ -115,6 +115,7 @@ function SeriesUnmemoized<T extends TimestampedValue>({ seriesConfig, seriesList
                   fillOpacity={config.fillOpacity}
                   getX={getX}
                   getY={getY}
+                  bandPadding={config.bandPadding}
```
Comment (VWSCoronaDashboard30) [inline packages/app/src/components/time-series-chart/components/series.tsx:118]: i don't see the `BarTrend` component being changed

Comment (VWSCoronaDashboard29) [reply packages/app/src/components/time-series-chart/components/series.tsx:118]: Parameter is already present in barTrend, optional

File: packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx (hunk 2 above)
Comment (VWSCoronaDashboard29) [reply packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx]: extracted to constant

File: packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx (hunk 3 above)
Comment (VWSCoronaDashboard29) [reply packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx]: extracted to constant

File: packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx [hunk 5]
```diff
@@ -156,110 +158,111 @@ const HospitalsAndCarePage = (props: StaticProps<typeof getStaticProps>) => {
...
+                      label: textNl.hospitals.chart_beds_occupied.legend_trend_label,
+                      color: colors.primary,
+                      bandPadding: gappedBarBandPaddingOverride,
+                    },
+                  ]}
+                  dataOptions={{
+                    timespanAnnotations: [
+                      {
+                        start: data.hospital_lcps.values[0].date_unix,
+                        end: new Date('1 June 2020').getTime() / 1000,
```
Comment (VWSCoronaDashboard30) [inline packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx:202]: This is non ticket related. I think we should look into this as it's a hard coded date

**Review by VWSCoronaDashboard30 — APPROVED** [review]
Well done!

File: packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx (hunk 5 above)
Comment (VWSCoronaDashboard29) [reply packages/app/src/pages/landelijk/ziekenhuizen-in-beeld.tsx:202]: Created a ticket COR-1879

**Pull request merged**